### PR TITLE
Add Feisar freestyle rates.txt

### PR DESCRIPTION
**Pull request by AndroidGX**

### CODE REVIEW

File: presets/4.3/rates/Feisar_freestyle.txt [hunk 1]
```diff
@@ -0,0 +1,28 @@
+#$ TITLE: Feisar freestyle rates
+#$ FIRMWARE_VERSION: 4.3
+#$ CATEGORY: RATES
+#$ STATUS: OFFICIAL
```
Comment (limonspb) [inline presets/4.3/rates/Feisar_freestyle.txt:4]: please make it a community one.

**Review by limonspb — CHANGES_REQUESTED** [review]

File: presets/4.3/rates/Feisar_freestyle.txt [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+#$ AUTHOR: Feisar
+#$ DESCRIPTION:
+#$ DESCRIPTION: [ Feisar freestyle rates ]
+#$ DESCRIPTION:
+#$ DESCRIPTION: Ideal for smooth freestyle
+#$ DESCRIPTION:
+#$ DESCRIPTION: (last preset update : 2022-02-13)
+#$ DESCRIPTION:
+#$ INCLUDE: presets/4.3/rates/defaults.txt
+
```
Comment (limonspb) [inline presets/4.3/rates/Feisar_freestyle.txt:15]: add `#$ DISCUSSION: https://github.com/betaflight/firmware-presets/pull/184`

**Review by limonspb — CHANGES_REQUESTED** [review]

**Comment by sugaarK** [comment]
@AndroidGX  wana add some screen shots to the description so people can get a visual idea of your setup like the other rate presets?

**Comment by limonspb** [comment]
plz don't forget to squash commits in one :)

**Comment by AndroidGX** [comment]
> 

Sure ! good idea :) 
Here is my screenshot : 

betaflight-configurator_gp41cTz3wm

**Review by limonspb — APPROVED** [review]

**Pull request merged**